### ISSUE TITLE
Checkbox choices aren't being rendered after using fromJSON?

### ISSUE THREAD

**Issue by leemi** [surveyjs/survey-library#3331]
### Are you requesting a feature, reporting a bug or asking a question?
I am reporting a bug. Thanks!

### What is the current behavior?
![image](https://user-images.githubusercontent.com/908639/133335474-3c0afb9a-3360-419d-a739-25ca9c859245.png)

### What is the expected behavior?
![image](https://user-images.githubusercontent.com/908639/133335502-d930dbb5-6a72-4431-8703-4d876b463d98.png)

### How would you reproduce the current behavior (if this is a bug)?
Load a page with choices into a PageModel via `fromJSON`

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: https://codesandbox.io/s/modern-tdd-t2yvh?file=/src/SurveyComponent.jsx

Test code

```javascript
import React, { Component } from "react";

import * as Survey from "survey-react";

import "survey-react/modern.css";
import "./index.css";

Survey.StylesManager.applyTheme("modern");

class SurveyComponent extends Component {
  render() {
    const json = {
      questions: [
        {
          type: "checkbox",
          name: "car",
          title: "What car are you driving?",
          isRequired: true,
          hasNone: true,
          colCount: 4,
          choices: [
            "Ford",
            "Vauxhall",
            "Volkswagen",
            "Nissan",
            "Audi",
            "Mercedes-Benz",
            "BMW",
            "Peugeot",
            "Toyota",
            "Citroen"
          ]
        }
      ]
    };
    const survey = new Survey.Model(json);

    // Serialize Page to JSON
    const pageModel = survey.pages[0];
    const pageJSON = pageModel.toJSON();

    // Remove Page
    survey.removePage(pageModel);

    // Re-add Page
    const pg = survey.addNewPage();
    pg.fromJSON(pageJSON);
    console.log(pageJSON);

    return <Survey.Survey model={survey} />;
  }
}

export default SurveyComponent;
```

### Specify your

* browser: Chrome 
* browser version: 93.0.4577.63 (Official Build)
* surveyjs platform (angular or react or jquery or knockout or vue): React
* surveyjs version: 1.8.65

**Comment by andrewtelnov** [comment]
@leemi I will take a look later.
As a work-around, please use the following code from `pg.fromJSON(pageJSON);`
`survey.endLoadingFromJson();`.

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
@leemi The better work-around would be `pg.onSurveyLoad();`.
I have fixed the issue by commit above, by adding a new unit test.
Unfortunately, this fix will likely go to v1.8.67 that we will release on next week and it will not in v1.8.66 that we are going to release shortly.

Thank you,
Andrew
